SSH TunnelBuilder/Models/ViewController.swift

```diff
@@ -43,6 +47,20 @@ class ViewController: NSViewController, NSComboBoxDataSource {
         self.view.window?.title = "SSH TunnelBuilder"
     }
 
+    @objc private func refresh() {
+        viewModel.refresh { error in
+            if let error = error {
+                let alert = NSAlert()
```

Comment:
Alert box her burde være i en egen funksjon. Kan lage en statisk klasse for dette.

class MyUtils
{
  static func ShowAlertBox( message: String, alertStyle: NSAlert.Style )
  {
              DispatchQueue.main.async {
                    let alert = NSAlert()
                    alert.alertStyle = alertStyle            
                    alert.messageText = message
                    alert.addButton(withTitle: "OK")
                    alert.runModal()
                    return
                }
  }
}

Reply:
Coolio - Added the class and function as suggested